app/serializers/webfinger_serializer.rb

```diff
@@ -29,7 +29,9 @@ def links
         { rel: 'http://webfinger.net/rel/profile-page', type: 'text/html', href: short_account_url(object) },
         { rel: 'self', type: 'application/activity+json', href: account_url(object) },
         { rel: 'http://ostatus.org/schema/1.0/subscribe', template: "#{authorize_interaction_url}?uri={uri}" },
-      ]
+      ].tap do |x|
+        x << { rel: 'http://webfinger.net/rel/avatar', type: object.avatar.content_type, href: full_asset_url(object.avatar_original_url) } if object.avatar.present? && object.avatar.content_type.present?
```

Comment:
Do we specifically not want this in place for the instance actor case? or was this just a first pass to get it in place at all?

Reply:
i'm not sure of the exact data structure for the instance actor object. would that even have the .avatar and .avatar_original_url fields as well?

Reply:
It's just an `Account` record, but with a special database ID, and special behavior within the app.

I was mostly just curious if you excluded on purpose ... thinking about it, I suspect that even though it could have attachments, its unlikely in practice actually to have them. This is probably fine where it is.

Reply:
i think it's a cool idea might as well! I also simplified `links` in the process https://github.com/mastodon/mastodon/pull/26558/commits/0671f8d78d8821339fb6beba92972dad3638051b